Contact Form

Kshitij841 opened the pull request:
css in contact.css, html in contact.html

rajatgupta24 reviewed (changes requested):
Hey @Kshitij841, don't create another **contact.html**, just create a form in the **index.html**
& try to resolve the conflicts

Kshitij841 commented:
> Hey @Kshitij841, don't create another **contact.html**, just create a form in the **index.html**
> & try to resolve the conflicts

Removed old contact form from index.html, added new contact form, css in contact.css

Kshitij841 commented:
Contact form added in index.html
css in footer.css
-border-radius: 15% 85% 79% 21% / 100% 4% 96% 0%; experimenting this, alternative: border-radius: 1rem; [line 13 footer.css]
